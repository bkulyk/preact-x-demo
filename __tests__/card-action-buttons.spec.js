import React from 'react';
import CardActionButtons from '@bitchin/card-action-buttons';
import { shallow } from './enzyme';

it('renders correctly, it is pretty much just a div', () => {
  expect(shallow(<CardActionButtons className="custom" x-data="my-data">...</CardActionButtons>))
    .toMatchSnapshot();
});
