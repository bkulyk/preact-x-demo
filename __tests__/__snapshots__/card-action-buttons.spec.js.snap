// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly, it is pretty much just a div 1`] = `
<div
  className="mdc-card__action-buttons custom"
  x-data="my-data"
>
  ...
</div>
`;
